bokehjs/src/lib/core/selection_manager.ts

```diff
@@ -26,8 +25,7 @@ export class SelectionManager extends HasProps {
     this.inspectors = {}
   }
 
-  select(renderer_views: RendererView[], geometry: Geometry, final: boolean, append: boolean = false): boolean {
-    // divide renderers into glyph_renderers or graph_renderers
+  _split_renderer_views_by_type(renderer_views: RendererView[]): {glyph_renderer_views: GlyphRendererView[], graph_renderer_views: GraphRendererView[]} {
```

Comment:
@mattpap left this comment on the last iteration: "You should also return an interface instead of a mapping, unless you can't (or won't be in future) predict what keys are returned." Was this what you meant by returning an interface?

Reply:
I believe so, yes